xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -283,7 +282,7 @@ bool CGUIDialogVideoInfo::OnAction(const CAction &action)
   return CGUIDialog::OnAction(action);
 }
 
-void CGUIDialogVideoInfo::SetUserrating(int userrating)
+void CGUIDialogVideoInfo::SetUserrating(int userrating) const
```

Comment:
Well I actually wondered about that too yesterday, so I did try to set an userrating and it seemed to did work without problems. I will try again just in case I missed it going wrong.